default.nix

```diff
@@ -70,8 +70,8 @@ let
             input_file="$dat_name"
           fi
         fi
-        ddlog -i "$program" -L '${builtins.toPath ./lib}'
-        (cd "''${program_noext}_ddlog" && cargo build --release)
```

Comment:
There are a couple of problems with debug builds in Rust: they are 10x slower (literally) and also take 10x more disk space than release builds. The release build is indeed slow, but after a recent refactoring only the first build is really slow; subsequent rebuilds usually take a few seconds, so I would stick to release builds here.

Reply:
indeed, rebuild with `--release` is only a few seconds longer than with `--debug` on a multicore machine. And disk space usage is greatly reduced. I'll revert back